.github/workflows/ci.yml

```diff
@@ -61,43 +61,21 @@ jobs:
           PATH=$PATH:main/tools/spell
           # Make sure that the portable directory is not included in the spellcheck.
           sed -i 's/find $DIRNAME/find $DIRNAME -not -path '*portable*'/g' main/tools/spell/find-unknown-comment-words
-          find-unknown-comment-words --directory tcp/ --lexicon tcp/.github/lexicon.txt
+          find-unknown-comment-words --directory tcp/ --lexicon tcp/lexicon.txt
           if [ "$?" = "0" ]; then
             exit 0
           else
             exit 1
           fi
 
   formatting:
-    # Use only 18.04 since we want the uncrustify version to
-    # be 0.66.1_f to ensure proper formatting.
     runs-on: ubuntu-20.04
-    container: ubuntu:18.04
     steps:
-      - uses: actions/checkout@v3
-      - name: Install Uncrustify
-        run: apt-get update && apt-get install uncrustify
-      - name: Run Uncrustify
-        run: |
-          uncrustify --version
-          find . -iname "*.[hc]" -exec uncrustify --check -c tools/uncrustify.cfg {} +
-          if [ "$?" = "0" ]; then
-            exit 0
-          else
-            echo -e "\033[31;1;43mFormatting check (using Uncrustify) failed...\033[0m"
-            echo -e "\033[32;3mTo have the code uncrustified for you, please comment '/bot run uncrustify' (without the quotes) on the Pull Request.\033[0m"
-            exit 1
-          fi
-      - name: Check For Trailing Whitespace
-        run: |
-          set +e
-          grep --exclude="README.md" -rnI -e "[[:blank:]]$" .
-          if [ "$?" = "0" ]; then
-            echo "Files have trailing whitespace."
-            exit 1
-          else
-            exit 0
-          fi
+      - uses: actions/checkout@v2
```

Comment:
This is probably a nit pick but maybe we want to use `actions/checkout@v3`? It uses a more up-to-date version of Node.

Reply:
Fully agree with you, but all of our repos are using the checkout@v2 where I would prefer to do that change across all of them at once compared to individual repos.